### PR TITLE
Remove ONNX BatchNorm(12) test and converter.

### PR DESCRIPTION
**Pull request by codemzs**
Pursuant to https://github.com/onnx/onnx/pull/2750 we must remove PyTorch ONNX exporter related changes to BatchNorm(12) that were introduced as part of https://github.com/pytorch/pytorch/pull/35567. This change is also needed to unblock ONNX [BUILD CI failures](https://circleci.com/gh/onnx/onnx/4629?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) caused by PyTorch/Caffe2 tests.

### CODE REVIEW

**Comment by codemzs** [comment]
@houseroad can you please review this?

**Review by lara-hdr — APPROVED** [review]
@houseroad BatchNorm was reverted from ONNX opset 12.
Could you please merge this change? thanks

**Comment by codemzs** [comment]
@houseroad can you please restart the failing builds, seems like an infra issue?

**Review by houseroad — APPROVED** [review]

**Comment by codemzs** [comment]
@houseroad Why was this PR closed?

**Comment by codemzs** [comment]
@houseroad Thank you very much for reviewing and merging my PR!
